app/src/main/java/edu/mills/cs115/AZCategoryAdapter.java

```diff
@@ -8,28 +8,52 @@
 import android.view.ViewGroup;
 import android.widget.TextView;
 
-class CategoryAdapter extends RecyclerView.Adapter<CategoryAdapter.ViewHolder> {
-    private final static String TAG = "CategoryAdapter";
+/**
+ * AZCategoryAdapter implements the adapter for the RecyclerView in AZCategoryFragment.
+ */
```

Comment:
I noticed Ellen added this line to her class files:
* @author Ellen Spertus (ellen.spertus@gmail.com)